<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallStat</class>
 <widget class="QDialog" name="CallStat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Получить данные</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>417</width>
       <height>78</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3" rowspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>InfoBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="callCounter">
        <property name="text">
         <string>CallCounter:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="limitsCell">
        <property name="text">
         <string>OnLimitsOneCell:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDateTimeEdit" name="over"/>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="cell"/>
   </item>
   <item row="1" column="0">
    <widget class="QDateTimeEdit" name="begin"/>
   </item>
   <item row="0" column="1" rowspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>46</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
